build_tools/run.py

```diff
@@ -22,7 +22,9 @@ def __init__(self, retcode, output):
         self.output = output
 
 def _to_str(x):
-    if isinstance(x, (bytes, bytearray)):
+    if x is None:
+        x = ''
```

Comment:
What function returned a None?

Reply:
If you specify `stdout = None`, like we do in the `env_command` function, `process.communicate()[0]` will return `None` instead of the standard output string.